public/view_lecture_teams.php

```diff
@@ -72,11 +72,58 @@
                 </table>
 
         <?php } ?>
-        </tbody>
-    </table>
     <?php } else { ?>
         <blockquote>No teams found for this course.</blockquote>
     <?php }?>
 
     <br><a href="create_team.php?id=<?php echo $lecture_page_id ?>">Create new teams</a> 
-<?php } ?>
\ No newline at end of file
+<?php }else {
+    try {
+        $all_student_team = TeamMember::includes(['teams' => 'lectures'])->where(array('user_id' => get_users_id(),));
+    
+    } catch (PDOException $error) {
+        echo "<br>" . $error->getMessage();
+    }
```

Comment:
hmm ok i have a différent proposal to find the team that the student is in,
```php
$student_team = Team::joins(["team_members"])->where(["user_id"=>get_users_id(), "lecture_id"=>$lecture_page_id]))
```

Reply:
its a lot simpler and easier to understand too

Reply:
doesn't work creates an error

Reply:
done